apier/v2/attributes.go

```diff
@@ -18,34 +18,23 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>
 
 package v2
 
-/*
 import (
 	"github.com/cgrates/cgrates/engine"
 	"github.com/cgrates/cgrates/utils"
 )
 
 //SetAttributeProfile add/update a new Attribute Profile
-func (apierV1 *ApierV2) SetAttributeProfile(alsPrf *engine.AttributeProfile, reply *string) error {
-	if missing := utils.MissingStructFields(alsPrf, []string{"Tenant", "ID"}); len(missing) != 0 {
+func (apierV1 *ApierV2) SetExternalAttributeProfile(extAlsPrf *engine.ExternalAttributeProfile, reply *string) error {
```

Comment:
Please rename the API SetAttributeProfile. This is the reason wee have added it directly inside v2